modules/api_docs/test/TestPlan.md

```diff
@@ -0,0 +1,53 @@
+# API Docs module Test Plan
+
+##  Overview
+
+The API docs modules takes the api descriptor of LORIS's modules and creates an 
```

Comment:
Why is the module overview in the test plan and not the README?

Reply:
Because it help the tester to understand what he/she is testing.
It can be added to the README if you want.